emoji/src/main/java/com/vanniktech/emoji/EmojiImageView.java

```diff
@@ -88,7 +88,7 @@ public EmojiImageView(final Context context, final AttributeSet attrs) {
     }
   }
 
-  public void setEmoji(@NonNull final Emoji emoji) {
+  void setEmoji(@NonNull final Emoji emoji) {
```

Comment:
This is not supposed to be overwritten, the class itself is only `public`, because of the package structure (it's used in some listeners, which reside in a different package). For that reason it has the `@RestrictTo(LIBRARY)` annotation.